### ISSUE TITLE
How to synthesize audio from checkpoint?

### ISSUE THREAD

**Issue by mligema** [NVIDIA/tacotron2#261]
Finally I got the _python -m multiproc train.py etc..._  to work. Simple question: How do I now synthesize audio form a specific checkpoint?

In github.com/keithito/tacotron it was pretty simple: _python eval.py --checkpoint 'path_to_checkpoint_', but here I don't see any documentation on generating wav's.

**Comment by Clement-Hui** [comment]
You can check out the inference.ipynb . It provides an easy way to inference the audio, but you have to download pretrained waveglow module or code your own way of making the spectrogram output to audio, maybe via wavenet vocoder.

**Comment by Yeongtae** [comment]
@Clement-Hui 
Use https://github.com/Yeongtae/tacotron2/blob/master/inference.py

**Comment by mligema** [comment]
@Yeongtae thanks! It is generating silent wavs. Any suggestions? Here's my melspectrogram
![sentence_0](https://user-images.githubusercontent.com/10670464/64536458-dc9a6c00-d321-11e9-86b3-eadc7aeab6aa.png)

It could be that my training hparams are the problem.

**Comment by Clement-Hui** [comment]
How many steps did you train? Model should converage around 10k steps.

**Comment by mligema** [comment]
95000 steps

**Comment by Clement-Hui** [comment]
Please refer to #254 and #256. In short, you should try training the model with the English pretrained model as initial weights. Also, you should set attention dropout to a higher value like 0.4. Also, the attention should be a diagonal line across the graph

**Comment by Clement-Hui** [comment]
Also you can try using the original inference.ipynb. maybe that would help.

**Comment by mligema** [comment]
So I trained with the Engish pretrained model, with dropout 0.4, to step 35000. Then I synthesized this voice using the inference.ipynb, but it sounds far too "rough" https://www.dropbox.com/s/1fltkjplbn5dlsb/step35000.mp3?dl=0
![index](https://user-images.githubusercontent.com/10670464/65410447-0dcf6d80-ddf3-11e9-8fc2-86656fa860d6.png)

Any ideas how to improve the quality of synthesized sound?

**Comment by Yeongtae** [comment]
> So I trained with the Engish pretrained model, with dropout 0.4, to step 35000. Then I synthesized this voice using the inference.ipynb, but it sounds far too "rough" https://www.dropbox.com/s/1fltkjplbn5dlsb/step35000.mp3?dl=0
> ![index](https://user-images.githubusercontent.com/10670464/65410447-0dcf6d80-ddf3-11e9-8fc2-86656fa860d6.png)
> 
> Any ideas how to improve the quality of synthesized sound?

Test my master branch. It use preprocessing for audio dataset and nomalization for melspectrogram.
https://github.com/Yeongtae/tacotron2

**Comment by Clement-Hui** [comment]
What is the data used? Maybe the data does not have enough samples

**Comment by mligema** [comment]
@Clement-Hui my dataset consists of 3277 wav files, 1.6GB, single-channel, 22050Hz, 352 kbps, and they are between 5-20 sec long.
Okey @Yeongtae I will try train a model with your fork tacotron2, I'll let you know if I have results.

**Comment by Yeongtae** [comment]
In my master branch, there is no monitoring metric. It is in the monitoring metric branch.

**Comment by Clement-Hui** [comment]
What is the method used to convert Mel spectrogram to wave? Is it waveglow or others? Try using wavenet vocoder.

**Comment by rafaelvalle** [comment]
Closing due to inactivity.
